core/db/entities.py

```diff
@@ -527,3 +527,12 @@ class ProjectFinanceChange(BaseModel):
             "programme_junction_id",
         ),
     )
+
+
+class GeospatialDim(BaseModel):
+    """Stores Geospatial information mapped to postcodes."""
+
+    __tablename__ = "geospatial_dim"
+
+    postcode_prefix = sqla.Column(sqla.String(), nullable=False, unique=True)
```

Comment:
probably could actually limit this in length? Maybe 4 chars? (check if there are any viable postcodes that start with 5 chars, but I don't think so)